source/server/config/http/grpc_web.cc

```diff
@@ -0,0 +1,30 @@
+#include "server/config/http/grpc_web.h"
+
+#include "common/grpc/grpc_web_filter.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+
+HttpFilterFactoryCb GrpcWebFilterConfig::tryCreateFilterFactory(HttpFilterType type,
```

Comment:
Please add a test for the filter config loading in https://github.com/lyft/envoy/blob/master/test/server/config/http/config_test.cc